.github/workflows/verify-transpiled-code.yaml

```diff
@@ -0,0 +1,35 @@
+name: 'verify-transpiled-code'
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          yarn install
+          mv dist/ source-dist
+          yarn run all
+
+      - name: Run integrity check
+        run: |
+          #!/bin/bash
+
+          hash_function="sha256sum"
+
+          for file in dist/*
+          do
+                  source_file=${file/dist/source-dist}
```

Comment:
I'm a little bit slow today...how is this getting the different source files?

Reply:
Yesterday I discovered [native string replacement in Bash](https://linuxhandbook.com/replace-string-bash/)!

I'll add a comment clarifying it though because maybe it's not well known